addons/debugger/icons/debug.svg

```diff
@@ -1 +1 @@
-<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg"><g stroke="#4C97FF" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"><path style="fill:#575e75;fill-opacity:1;stroke:none;stroke-width:1;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1;paint-order:markers stroke fill" d="M4 3.5c-.831 0-1.5.669-1.5 1.5v10c0 .831.669 1.5 1.5 1.5h12c.831 0 1.5-.669 1.5-1.5V5c0-.831-.669-1.5-1.5-1.5Zm0 1h12c.277 0 .5.223.5.5v10c0 .277-.223.5-.5.5H4a.499.499 0 0 1-.5-.5V5c0-.277.223-.5.5-.5Z" stroke="none"/><path d="m5.417 7.5 3.75 2.5-3.75 2.5m5 0h4.166" style="fill:none;stroke:#8f8f8f;stroke-width:1;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"/></g></svg>
\ No newline at end of file
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg"><g stroke="#855cd6" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"><path style="fill:#575e75;fill-opacity:1;stroke:none;stroke-width:1;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1;paint-order:markers stroke fill" d="M4 3.5c-.831 0-1.5.669-1.5 1.5v10c0 .831.669 1.5 1.5 1.5h12c.831 0 1.5-.669 1.5-1.5V5c0-.831-.669-1.5-1.5-1.5Zm0 1h12c.277 0 .5.223.5.5v10c0 .277-.223.5-.5.5H4a.499.499 0 0 1-.5-.5V5c0-.277.223-.5.5-.5Z" stroke="none"/><path d="m5.417 7.5 3.75 2.5-3.75 2.5m5 0h4.166" style="fill:none;stroke:#8f8f8f;stroke-width:1;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"/></g></svg>
```

Comment:
This stroke color isn't actually used anywhere in the icon, so changing it isn't necessary.